src/collection-list/__tests__/CollectionItem.test.jsx

```diff
@@ -79,4 +79,28 @@ describe('CollectionItem', () => {
       expect(wrapper.find(MetadataItem)).toHaveLength(0)
     })
   })
+
+  describe('when no heading URL is passed', () => {
+    beforeAll(() => {
+      wrapper = mount(
+        <CollectionItem
+          headingText={capitalProfileItem.headingText}
+          badges={capitalProfileItem.badges}
+          metadata={capitalProfileItem.metadata}
+        />
+      )
+    })
+
+    test('should render the component', () => {
+      expect(wrapper.find(CollectionItem).exists()).toBe(true)
+    })
+
+    test('should render the headingText', () => {
+      expect(wrapper.find('h3').text()).toBe('Mars Exports Ltd')
+    })
+
+    test('should not render the headingUrl', () => {
+      expect(wrapper.find('a').exists()).toBe(false)
```

Comment:
One more option to write this assetion:
```suggestion
      expect(wrapper.find('a')).toHaveLength(0)
```

Reply:
do we have a preference? I've used `exists()).toBe` higher up in the test file, so it is consistent. is one more performant/better than the other?

Reply:
I personally tend to use `expect(wrapper.find('a')).toHaveLength(0)` as it uses fewer characters and is easier to read.

Can't comment on the performance.